app.py

```diff
@@ -4,22 +4,56 @@
 from elasticsearch import Elasticsearch
 from summary_1.summary import body_summary
 from knn_indexing.index import knn_query
+
+
+import logging
+
+import logging.config
+
+# from logsetup import LOGGING_CONFIG
+
+logging.config.dictConfig()
+
+logger = logging.getLogger(__name__)
+
+import redis
+
 from modules.RateLimiter.src.request_handler import RateLimiter
 
+
 # INDEX_NAME = 'news'
 
 # ES = Elasticsearch([{'host' : 'localhost', 'port': 9200}])
 
 INDEX_NAME = 'knn_index'
 
-ES = Elasticsearch("https://admin:admin@localhost:9200", verify_certs=False, ssl_show_warn=False)
+ES = Elasticsearch("http://admin:admin@localhost:9200", verify_certs=False, ssl_show_warn=False)
 
 app = Flask(__name__)
 
 cors = CORS(app)
 
 app.config['CORS_HEADERS'] = 'Content-Type'
 
```

Comment:
**[Required, Merge Conflict]** You'll need to add in the exception wrapper here, merging in from the `main` branch should cover that